<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChessForm</class>
 <widget class="QWidget" name="ChessForm">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="2">
    <widget class="QFrame" name="frame_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>500</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbl2">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>91</width>
        <height>81</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>130</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_blackHandOut">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
       </font>
      </property>
      <property name="text">
       <string>交出下子权力</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_blackGameOver">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>350</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
       </font>
      </property>
      <property name="text">
       <string>认 输</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_b">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>230</y>
        <width>72</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
       </font>
      </property>
      <property name="text">
       <string>剩余时间：</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_blackLeftTime">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>220</y>
        <width>111</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame_3">
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>70</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>120</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QComboBox" name="cbox_item">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>20</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
       </font>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>20</y>
        <width>295</width>
        <height>30</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btn_pvp">
         <property name="font">
          <font>
           <family>楷体</family>
          </font>
         </property>
         <property name="text">
          <string>人人对战</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QPushButton" name="btn_pvc">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>20</y>
        <width>281</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
       </font>
      </property>
      <property name="text">
       <string>人机对战</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>15</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>请选择谁先下子：</string>
      </property>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QGridLayout" name="gridLayout"/>
   </item>
   <item row="0" column="0">
    <widget class="QFrame" name="frame">
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QLabel" name="lbl1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>101</width>
        <height>91</height>
       </rect>
      </property>
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNumber1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>101</width>
        <height>61</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Panel</enum>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_whiteHandOut">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>交出下子权力</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btn_whiteGameOver">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
       </font>
      </property>
      <property name="text">
       <string>认 输</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbl_w">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>240</y>
        <width>72</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
       </font>
      </property>
      <property name="text">
       <string>剩余时间：</string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcd_whiteLeftTime">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>楷体</family>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
